llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1823,6 +1823,10 @@ def TuneConditionalCompressedMoveFusion
 def HasConditionalMoveFusion : Predicate<"Subtarget->hasConditionalMoveFusion()">;
 def NoConditionalMoveFusion  : Predicate<"!Subtarget->hasConditionalMoveFusion()">;
 
+def TuneHasThrottledVecFP64
```

Comment:
How about SingleElementVecFP64 instead of some arbitrary notion of "throttling"?